.github/workflows/macos-deploy.yml

```
@@ -0,0 +1,92 @@
+on: [push]
+
+jobs:
+  buildApp:
+    name: safari build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: setup env
+        id: node-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:2:41: Quote this to prevent word splitting

(shellcheck)